test/client/spec/victory-shared-events/victory-shared-events.spec.js

```diff
@@ -0,0 +1,129 @@
+/**
+ * Client tests
+ */
+import { curry, flow, forEach } from "lodash";
+import { mount } from "enzyme";
+import React from "react";
+import { addEvents } from "src/index";
+import VictorySharedEvents from "src/victory-shared-events/victory-shared-events";
+import { MockVictoryComponent, MockDataComponent } from "../mock-components";
+
+describe("components/victory-shared-events", () => {
+  const EventedMockVictoryComponent = addEvents(MockVictoryComponent);
+
+  const findVictoryComponentByName = curry((name, component) => {
```

Comment:
Yeah, `shallow` rendering wouldn't work in this case since we need to trigger events on `children`, and they aren't rendered. `mount` will require the workaround in order to access the custom component via props.